### ISSUE TITLE
Use optimizer for planning backflip trajectory

### ISSUE THREAD

**Issue by jviereck** [machines-in-motion/kino-dynamic-opt#7]
At this point, it is not possible to optimize a backflip motion with the optimizer on the solo robot.

This issue is about tracking the required modifications / extensions to the optimizer to make this possible.

**Comment by jviereck** [comment]
Some points include:
* better endeffector trajectory planning
* have ability to specify desired base angle in the inverse kinematics along the trajectory

**Comment by majidkhadiv** [comment]
This is true, I will try to implement more delicate (while yet dynamic) motions, e. g. backflip and jumping on the box, using kino-dyn planner. But I think it needs a huge amount of work in terms of landing control and dynamically-consistent end-effector trajectory generation. I have some ideas to do that, but let's see how much time this will take... I will be devoting most of my time to this in the rest of this year and early next year...
